docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/index.md

```diff
@@ -350,6 +350,102 @@ dbt run --selector snowplow_web
 
 
 </TabItem>
-</Tabs>
+<TabItem value="ecommerce" label="Snowplow Ecommerce">
+
+#### 1. Adding the `selector.yml` file
+
+Within the packages we have provided a suite of suggested selectors to run and test the models within the package together with the ecommerce model. This leverages dbt's [selector flag](https://docs.getdbt.com/reference/node-selection/syntax). You can find out more about each selector in the [YAML Selectors](#yaml-selectors) section.
+
+These are defined in the `selectors.yml` file ([source](https://github.com/snowplow/dbt-snowplow-ecommerce/blob/main/selectors.yml)) within the package, however in order to use these selections you will need to copy this file into your own dbt project directory. This is a top-level file and therefore should sit alongside your `dbt_project.yml` file. If you are using multiple packages in your project you will need to combine the contents of these into a single file.
+
+#### 2. Check source data
+
+This package will by default assume your Snowplow events data is contained in the `atomic` schema of your [target.database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile), in the table labeled `events`. In order to change this, please add the following to your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__atomic_schema: schema_with_snowplow_events
+    snowplow__database: database_with_snowplow_events
+    snowplow__events_table: table_of_snowplow_events
+```
 
+:::info Databricks only
 
+Please note that your `target.database` is NULL if using Databricks. In Databricks, schemas and databases are used interchangeably and in the dbt implementation of Databricks therefore we always use the schema value, so adjust your `snowplow__atomic_schema` value if you need to.
+
+:::
+
+#### 3. Enable desired modules
+
+The ecommerce package has the option to join in data from the following 1 Snowplow module:
+
+- App Errors module -- Details relating to app errors that occur during sessions
+
+By default this module is **disabled** in the ecommerce package. Assuming you have the enrichments turned on in your Snowplow pipeline, to enable the module within the package please modify the following in your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__enable_app_errors_module: true
+```
+
+#### 4. Filter your data set
+
+You can specify both `start_date` at which to start processing events and the `app_id`'s to filter for. By default the `start_date` is set to `2020-01-01` and all `app_id`'s are selected. To change this please add/modify the following in your `dbt_project.yml` file:
+
+```yml
+# dbt_project.yml
+...
+vars:
+  snowplow_ecommerce:
+    snowplow__start_date: 'yyyy-mm-dd'
+    snowplow__app_id: ['my_app_1','my_app_2']
+```
```

Comment:
❓ Is it worth adding `snowplow__ecommerce_event_names` here given if it's something you'd expect most people to provide?